assembl/static2/js/app/reducers/adminReducer/adminSections.js

```diff
@@ -0,0 +1,59 @@
+// @flow
+import { combineReducers } from 'redux';
+import type ReduxAction from 'redux';
```

Comment:
What is ReduxAction? I don't see it in the lib except in a test: https://github.com/reactjs/redux/search?utf8=%E2%9C%93&q=ReduxAction&type=

Reply:
From what I remember, I had problems trying to type reducers and found this in code with types on reducers. I saw it as a wrapper that says it is any of the redux "default" actions (`@@INIT` and so) or any type in `Action`
According to flow doc https://flow.org/en/docs/react/redux/ it is not necessary (anymore?) so I will try to remove this after the merges. Thanks for pointing that out

Reply:
https://github.com/reactjs/redux/issues/290